<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="angelohb" failures="3" tests="6" name="BankingApp Test" time="71.412" errors="0" timestamp="2020-01-10T11:40:38 PET">
  <testcase classname="com.inetbanking.testCases.TC_LoginTest_001" name="loginTest" time="2.556">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginTest_001.loginTest(TC_LoginTest_001.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginTest -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="12.316"/>
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="4.859">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="4.948">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:32)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.inetbanking.testCases.TC_LoginDDT_002" name="loginDDT" time="11.16"/>
  <testcase classname="com.inetbanking.testCases.TC_AddCustomerTest_003" name="addNewCustomer" time="16.227"/>
</testsuite> <!-- BankingApp Test -->
